docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -793,6 +793,5 @@ Interface
 .. _`PBKDF2`: https://en.wikipedia.org/wiki/PBKDF2
 .. _`scrypt`: https://en.wikipedia.org/wiki/Scrypt
 .. _`key stretching`: https://en.wikipedia.org/wiki/Key_stretching
-.. _`HKDF`:
-.. _`RFC 5869`: https://tools.ietf.org/html/rfc5869
+.. _`HKDF`: https://en.wikipedia.org/wiki/HKDF
```

Comment:
This changse the URL, before `HKDF` pointed at the RFC. I'm honestly not sure if that was intentional -- if you prefer that let me know.